lib/Constants.php

```diff
@@ -26,6 +26,20 @@
 use OCP\Share\IShare;
 
 class Constants {
+	/**
+	 * Constants for API Controllers
+	 */
+	public const API_BASE = '/api/{apiVersion}/';
+	public const API_V3_REQUIREMENTS = [
```

Comment:
Should we also have the version in the constant name? I mean if we some when go with v4 we would have to rename all places?
Maybe just

```suggestion
	public const API_REQUIREMENTS = [
```

Reply:
I would merge it like this for now because we need to remove the constants again for making the openapi-extractor work. So this name will only last for a short period of time